helper/assethelper/helper.go

```diff
@@ -0,0 +1,32 @@
+package assethelper
+
+import (
+	rice "github.com/GeertJohan/go.rice"
+)
+
+func loadStaticFile(filename string) (string, error) {
+	box, err := rice.FindBox("../../static")
```

Comment:
I wonder if we should test this on a different OS working as well.. Probably is, just wondering.